src/Maestro/Client/src/MaestroApi.cs

```diff
@@ -72,6 +72,7 @@ public partial class MaestroApi
         public static TokenCredential CreateApiCredential(
             string barApiBaseUri,
             bool disableInteractiveAuth,
+            string cachePath = "",
```

Comment:
Why not this?

```suggestion
            string? cachePath = null,
```

What happens if someone passes `""`?